### ISSUE TITLE
Terminate testing mongodb server

### ISSUE THREAD

**Issue by IhostVlad** [mikhail-angelov/mongo-unit#2]
Common use case for `mongo-unit` package is following: launch testing mongodb server, perform test suite execution and then terminate server. First and second items are done great, but third one now causes some problems.

If `mongo-unit` had not been terminated, test runner will hang forever. If try to terminate server by `testConnection.command({ shutdown: 1 })` command, then `unhandledRejection` fires, which refers on closed connection - which of course had been forcefully closed by stopping the server.  

Which is right way to dispose `mongo-unit` at `afterAll` test section? Example code here:
```
import mongoUnit from 'mongo-unit';
import { MongoClient } from 'mongodb';

describe('Test suite', () => {
    let testConnection;
    beforeAll(async () => {
        const connectionUrl = await mongoUnit.start({ dbName: 'admin' });
        testConnection = await MongoClient.connect(connectionUrl);
    });
    afterAll(async () => {
        await testConnection.dropDatabase();
        await testConnection.command({ shutdown: 1 });
        await testConnection.close();
        testConnection = null;
    });
    /* Here follows some test suite that uses testConnection  */
}
```

**Comment by mikhail-angelov** [comment]
Good question @IhostVlad 
this library does not have good solution for it for not
there is internal API 
makeSureOtherMongoProcessesKilled
I use it, to prevent run several mongod instances
I can make it public, but maybe it's better to make a new API for it
the problem with makeSureOtherMongoProcessesKilled - it depends on platform, it works on Linux and MacOS only

I'll make a dedicated API for it
mongo-unit.stop()
if you use mocha it should be run at the very end of test suite
e.g. 
`afterl( () => mongo-unit.stop());`
outside of `describe` section

**Comment by mikhail-angelov** [comment]
I've added stop API
mongo-unit@1.3.0

**Comment by IhostVlad** [comment]
Thanks! Will try in our unit tests! :)

**Comment by akoptelov** [comment]
@mikhail-angelov can you also clear the dbUrl var in the stop() function? Otherwise it is impossible to use mongo in different test suites as second call to start() just does nothing while the server is not actually running.

**Comment by mikhail-angelov** [comment]
done, thanx for PR

**Comment by akoptelov** [comment]
@mikhail-angelov, thanks. I guess you can close this one, if @IhostVlad doesn't mind.

**Comment by mikhail-angelov** [comment]
released mongo-unit@1.3.1
